<ui version="4.0" >
 <class>ExceptionDialogBase</class>
 <widget class="QDialog" name="ExceptionDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Repeating Appointment</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="lblMessage" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string comment="series refers to a number of subsequent appointments (e.g. every Tuesday at 2pm)" >&lt;qt>This appointment is part of a series. Select the part of the series you want to change below.&lt;/qt></string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkEarlier" >
     <property name="text" >
      <string comment="part of a series of appointments" >Earlier Appointments</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkSelected" >
     <property name="text" >
      <string comment="part of a series of appointments" >This Appointment</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkLater" >
     <property name="text" >
      <string comment="part of a series of appointments" >Later Appointments</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
